tailwind.config.js

```diff
@@ -105,7 +105,8 @@ module.exports = {
         'error-semi-transparent': 'var(--error-semi-transparent)',
       },
       boxShadow: {
-        '2xl': '0rem 0.25rem 2.75rem 0rem rgb(0 0 0 / 0.25)',
+        '2xl':
+          '0rem 0.25rem 2.75rem 0rem rgb(0 0 0 / 0.25), inset 0px 0px 1rem rgb(0 0 0 / 0.25)',
```

Comment:
I double checked where we use `shadow-2xl`, it's used for `primary Button` and for `gradient button`, both are fine with this modifier:
![image](https://user-images.githubusercontent.com/49658988/192341886-6e9e016a-8bd6-40af-ada8-a7f064374fcd.png)

https://user-images.githubusercontent.com/49658988/192342054-daac877b-3d97-4fd2-89b6-406f67586cf0.mp4